Module http-server is not installed

Hi, after installing the npm package, and running `remix-ide`, it could not find http-server node-module, had to install it manually.

Hi,I had the problem,too.
I solve it.

D:\fs\envs\nodejs2>npm install remix-ide -g
C:\Users\changshoumeng\AppData\Roaming\npm\remix-ide -> C:\Users\changshoumeng\AppData\Roaming\npm\node_modules\remix-ide\bin\remix-ide
+ remix-ide@0.5.2-alpha.7
added 1 package in 1.471s

D:\fs\envs\nodejs2>remix-ide
module.js:549
    throw err;
    ^

Error: Cannot find module 'http-server'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\changshoumeng\AppData\Roaming\npm\node_modules\remix-ide\bin\remix-ide:2:18)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

D:\fs\envs\nodejs2>npm-run http-server
'npm-run' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

D:\fs\envs\nodejs2>npm install http-server -g
C:\Users\changshoumeng\AppData\Roaming\npm\http-server -> C:\Users\changshoumeng\AppData\Roaming\npm\node_modules\http-server\bin\http-server
C:\Users\changshoumeng\AppData\Roaming\npm\hs -> C:\Users\changshoumeng\AppData\Roaming\npm\node_modules\http-server\bin\http-server
+ http-server@0.11.1
added 23 packages in 7.816s

D:\fs\envs\nodejs2>remix-ide
Starting Remix IDE at http://localhost:8080

--- a/package.json
+++ b/package.json
@@ -24,7 +24,6 @@
     "exorcist": "^0.4.0",
     "fast-async": "^6.1.2",
     "fast-levenshtein": "^2.0.6",
-    "http-server": "0.9.0",
     "javascript-serialize": "^1.6.1",
     "jquery": "^3.3.1",
     "js-base64": "^2.1.9",
@@ -55,6 +54,7 @@
     "yo-yoify": "^3.1.0"
   },
   "dependencies": {
+    "http-server": "0.9.0",
     "remixd": "git+https://github.com/ethereum/remixd.git"
   },
   "repository": {

